### DISCUSSION TITLE
OT: What external GPS receiver do you recommend?

### DISCUSSION THREAD

**Discussion by rugk** [OpenOrienteering/mapper#575] [General]
Sorry that this question is a bit off-topic, but in your doc [you say that external GPS receivers are recommend](https://github.com/OpenOrienteering/mapper-manual/blob/gh-pages/pages/android-requirements.md#sensors).
This is obvious, but what devices exactly can you recommend?

Because I can find many Bluetooth GPS receivers, which only have Bluetooth 2.0/2.1 support, but there are no ones with Bluetooth 4.0 support. (Bluetooth LE would of course be fantastic, but even normal Bluetooth 4.0 support is okay)
So can you recommend some devices?

**Comment by dg0yt** [comment]
I'm afraid there is not much innovation in that market, given that _some_ GPS is now included in virtually every mobile device. The external GPS receivers may still benefit from better antennas (less design trade-offs) and independent positioning (of the device).
